presto-product-tests/src/main/java/com/facebook/presto/tests/jdbc/JdbcTests.java

```diff
@@ -113,11 +112,13 @@ public void shouldExecuteQueryWithSelectedCatalogAndSchema()
     public void shouldSetTimezone()
             throws SQLException
     {
-        String timeZoneId = "Indian/Kerguelen";
-        connection.setTimeZoneId(timeZoneId);
-        try (Statement statement = connection.createStatement()) {
-            QueryResult result = queryResult(statement, "select current_timezone()");
-            assertThat(result).contains(row(timeZoneId));
+        if (connection instanceof PrestoConnection) {
```

Comment:
otherwise test should not be enabled for simba. So instead of if please use Precondition.checkState()